conf/app.routes

```diff
@@ -30,8 +30,8 @@ GET        /:arrivalId/keep-alive
 
 GET        /:arrivalId/cannot-send-unloading-remarks                                                                 controllers.CannotSendUnloadingRemarksController.onPageLoad(arrivalId: ArrivalId)
 
-GET        /:arrivalId/unloading-guidance/:messageId                                                                 controllers.UnloadingGuidanceController.onPageLoad(arrivalId: ArrivalId, messageId: String)
-POST       /:arrivalId/unloading-guidance/:messageId                                                                 controllers.UnloadingGuidanceController.onSubmit(arrivalId: ArrivalId, messageId: String)
+GET        /:arrivalId/unload-goods/:messageId                                                                       controllers.UnloadingGuidanceController.onPageLoad(arrivalId: ArrivalId, messageId: String, mode:Mode = NormalMode)
+POST       /:arrivalId/unload-goods/:messageId                                                                       controllers.UnloadingGuidanceController.onSubmit(arrivalId: ArrivalId, messageId: String, mode:Mode = NormalMode)
```

Comment:
Is mode needed here if we're not defining CheckMode routes?

Reply:
And similarly, do we need message ID in these URLs? Struggling to see what purpose they're serving. If it's the case that we need it somewhere so we can navigate back to a page that does require the message ID, then perhaps we should consider whether that one place should fetch the message ID using the arrival ID

Reply:
At the moment it looks like more and more places are going to need the message ID just to facilitate the one place where it's actually used

Reply:
refactored code.